Inventor_API/Models/Category.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Inventor_API.Models
+{
+    public class Category
+    {
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+
+        public List<FoodItem> FoodItems { get; set; } = new List<FoodItem>();
```

Comment:
Look into the difference between an interface and implementation.

Reply:
For more context she means IList and List